### PR TITLE
Add some documentation for elapsed_time

### PR DESCRIPTION
**Pull request by Jacobkg**
Follow-up to https://github.com/travis-ci/travis-tasks/pull/56

@BanzaiMan

### CODE REVIEW

File: user/notifications.md [hunk 1]
```diff
@@ -495,6 +496,14 @@ You can specify multiple channels as well.
...
 
+You also have the possibility to customize the message that will be sent to the channel(s) with a template:
+
+    notifications:
+      slack:
+        template:
+          - 'Build <%{build_url}|#%{build_number}> (<%{compare_url}|%{commit}>) of %{repository}@%{branch} by %{author} %{result} in %{elapsed_time}'
+          - '%{pull_request_url} %{commit_subject}'
+
 As always, it's recommended to encrypt the credentials with our
```
Comment (plaindocs) [inline user/notifications.md]: Couple of comments here:

* better to cut to the chase on line 499 "To customize the message"...
* please use triple backtick code sections as they have syntax highlighting

```
```json
```

**Comment by Jacobkg** [comment]
@plaindocs Thanks for the feedback. I decided to trim back this change to just include the added template variable.

**Comment by Jacobkg** [comment]
@plaindocs @BanzaiMan Any further comments?

**Comment by plaindocs** [comment]
Great, thanks!

**Pull request merged**